src/AccessibilityInsights.CI/AccessibilityInsights.CI.nuspec

```
@@ -1,15 +1,20 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <package>
   <metadata>
-    <id>AxeWindows</id>
+    <id>Axe.Windows</id>
     <version>$buildnumber$</version>
-    <title>AxeWindows Automation Library</title>
-    <summary>AxeWindows automation library</summary>
-    <description>This package provides the AxeWindows core library. This library can be accessed either via a .NET app, or from PowerShell via "Install-Module Axe.Windows.Automation.dll", followed by "Start-AxeWindows", "Invoke-Snapshot" and "Stop-AxeWindows"</description>
-    <authors>AxeWindows team</authors>
+    <title>axe-windows</title>
+    <summary>Automated accessibility analysis for Windows applications</summary>
+    <description>The package provides the assemblies necessary to automate accessibility testing for Windows applications using the included axe-windows rule set. Analysis can be run either via a .NET app, or from PowerShell via "Install-Module Axe.Windows.Automation.dll", followed by "Start-AxeWindows", "Invoke-Snapshot" and "Stop-AxeWindows"</description>
+    <ProjectURL>https://github.com/Microsoft/axe-windows</ProjectURL>
```

this needs to be camelCase e.g. projectUrl
```suggestion
    <projectUrl>https://github.com/Microsoft/axe-windows</projectUrl>
```